src/components/LocationSearch/useNominatimAutocomplete.ts

```diff
@@ -59,12 +75,12 @@ const useNominatimAutocomplete = (input: string | unknown[]) => {
       return;
     }
     setPlaces([]);
-  }, [input, setPlaces]);
+  }, [input]);
 
-  const getPlaceLatLng = ({ location }) => {
+  const getPlaceLatLng = (place: Place): Coordinates => {
     return {
-      lat: parseFloat(location.lat),
-      lng: parseFloat(location.lng),
+      lat: parseFloat(place.location.lat),
+      lng: parseFloat(place.location.lng),
```

Comment:
Why parse as float if you already know it's a `number`?

Reply:
good point - I've changed the type & removed these. instead parsing the nominatim result. another thing we can validate with zod potentially down the line